src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -463,16 +463,6 @@ protected static CSharpCompilationOptions WithNonNullTypes(CSharpCompilationOpti
             return (options ?? TestOptions.ReleaseDll).WithNullableContextOptions(nullableContextOptions);
         }
 
-        protected static string NonNullTypesOff()
```

Comment:
>protected static string NonNullTypesOff() [](start = 8, length = 41)

@cston Could you please hold on on merging this change. I am about to create a PR that will be affected by removal of these helpers.